lib/moodle_rb.rb

```diff
@@ -10,7 +10,7 @@
 require 'moodle_rb/users'
 
 module MoodleRb
-  def self.new(token, url)
-    Client.new(token, url)
+  def self.new(token, url, timeout = 600)
```

Comment:
How about we make this
```
def self.new(token, url, query_options = {})
```
And then `.merge(query_options)` in the httparty requests.

This would allow us to set timeout, or any httparty option.